### PR TITLE
Fixing bugs

### PR DESCRIPTION
**Pull request by FedorKam**

### CODE REVIEW

File: src/components/Job.tsx [hunk 1]
```diff
@@ -20,6 +25,23 @@ export function Job() {
...
+
+    file_input!.classList.add("job__file_active");
+    file_input_name!.innerHTML = file.name;
+  };
+
+  useEffect(() => {
+    let input = document.querySelector(".job__input_tel");
+    input!.addEventListener("input", mask, false);
+    input!.addEventListener("focus", mask, false);
+    input!.addEventListener("blur", mask, false);
```
Comment (inthedark122) [inline src/components/Job.tsx]: need to add `removeEventListener`

```
return () => {
    input!.removeEventListener("input", mask);
    input!.removeEventListener("focus", mask);
    input!.removeEventListener("blur", mask);
}
```

File: src/components/Job.tsx [hunk 2]
```diff
@@ -78,17 +100,18 @@ export function Job() {
           <div className="job__form">
             <img src={form_logo} className="job__formLogo" />
             <form>
-              <input type="text" placeholder="Имя" className="job__input" />
-              <input type="text" placeholder="Фамилия" className="job__input" />
-              <input type="tel" placeholder="Телефон" className="job__input" />
-              <input type="email" placeholder="Email" className="job__input" />
+              <input type="text" placeholder="Имя" className="job__input" required/>
+              <input type="text" placeholder="Фамилия" className="job__input" required/>
+              <input type="tel" placeholder="Телефон" className="job__input job__input_tel" required/>
```
Comment (inthedark122) [inline src/components/Job.tsx]: or will be better to use react events:

`<input type="tel" placeholder="Телефон" className="job__input job__input_tel" onInput={mask} onFoucs={mask} onBlur={mask} required/>`

File: src/components/Partner.tsx [hunk 3]
```diff
@@ -11,12 +11,23 @@ import {
...
   const [isVWAnimated, setIsVWAnimated] = useState(false);
 
   const [isPopUpVisible, setIsPopUpVisible] = useState(false);
 
+  useEffect(() => {
+    let input = document.querySelector(".partner__input_tel");
+    input!.addEventListener("input", mask, false);
+    input!.addEventListener("focus", mask, false);
+    input!.addEventListener("blur", mask, false);
+  });
```
Comment (inthedark122) [inline src/components/Partner.tsx]: make similar to `src/components/Job.tsx`

File: src/utils/tel.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        let range = elem.createTextRange();
+        range.collapse(true);
+        range.moveEnd("character", pos);
+        range.moveStart("character", pos);
+        range.select()
+    }
+}
+  
+export function mask(event:any) {
+    let matrix = "+7 (___) ___ ____",
```
Comment (inthedark122) [inline src/utils/tel.ts]: we are looking from different countries

not good idea to make the mask only for russian

Comment (inthedark122) [reply src/utils/tel.ts]: could be better to skip this validation and just to validate on base rules

We can check:
1. length of number
2. `+` at the start

**Review by inthedark122 — CHANGES_REQUESTED** [review]

File: src/utils/tel.ts (hunk 4 above)
Comment (FedorKam) [reply src/utils/tel.ts]: The length of number is different in different countries. Could we check + at the start and that phone length between 10 and 15?

Comment (inthedark122) [reply src/utils/tel.ts]: yes, this is mutch better

**Review by inthedark122 — APPROVED** [review]

**Pull request merged**